tools/webhook-router/Azure.Sdk.Tools.WebhookRouter.Administration/New-AzureDevOpsRuleCredentials.ps1

```diff
@@ -0,0 +1,18 @@
+$username = New-Guid
+$password = New-Guid
+$salt = New-Guid
+
+$basicAuthorizationHeaderValue = "$($username):$($password)"
```

Comment:
What is the script for? Is it just for your local development?

Reply:
I plan on having a module which anyone who needs to manage rules can use to do things like list rules, add rules and delete rules. But for now I needed a script that could generate the username, password, salt and hash. Rather than getting people to calculate it themselves they can use this script.